tests/test_plugins/simple_scenes.py

```diff
@@ -0,0 +1,29 @@
+from manim import *
+
+
+class SquareToCircle(Scene):
+    def construct(self):
+        square = Square()
+        circle = Circle()
+        self.play(Transform(square, circle))
+
+
+class FunctionLikeTest(Scene):
+    def contruct(self):
+        assert "FunctionLike" in globals()
+        a = FunctionLike()
+        self.play(FadeIn(a))
+
+
+class NoAllTest(Scene):
+    def construct(self):
+        assert "test_plugin" in globals()
+        a = test_plugin.NoAll()
+        self.play(FadeIn(a))
+
+
+class WithAllTest(Scene):
+    def construct(self):
+        assert "WithAll" in globals()
```

Comment:
How are you guaranteeing that WithAll is in globals?

Reply:
Oh I just realized this file is not processed by pytest because it doesn't have "test" in its name. Makes sense!